files/install-deps-worker.yaml

```diff
@@ -8,11 +8,16 @@
       - python3-ipdb  # for easy debugging
       - nss_wrapper  # openshift anyuid passwd madness
       - python3-celery # (#107)
+      - rebase-helper
       - python3-redis
       - redis  # redis-cli for debugging
       - origin-clients # for sandcastle
       - python3-kubernetes # for sandcastle
       state: present
+  - name: Install packit
+    pip:
+      name: git+https://github.com/packit-service/packit.git
+      executable: pip3
```

Comment:
NACK.
If you want newer `packit` in `prod`, move [packit's stable branch](https://github.com/packit-service/packit/tree/stable) and wait for Docker Hub to do the rest (rebuild `docker.io/usercont/packit:prod` -> rebuild `docker.io/usercont/packit-service-worker:prod`)

Reply:
aha, so that's how we get it; okay, I'll revert that commit

we need some CD and image promotion